src/pages/donate.astro

```diff
@@ -23,21 +23,27 @@ import { resolvePath } from '@/utils/helpers.js'
         </h2>
         <p class="c-main-para">If you want to support our project and get rewards like early access to our development or your name listed on this page, you can back us! Here’s how:</p>
       </div>
-      <section class="container c-grid">
+
+      <section class="container c-grid c-mb-6">
         <div class="c-direct">
           <h2 class="c-sub-title">Direct Cryptocurrency Donation</h2>
           <Donate client:load/>
           <div class="c-or">or</div>
         </div>
         <div class="c-credit">
-          <h2 class="c-sub-title">Credit/Debit Card Donation</h2>
-          <iframe class="c-donation-box" src="https://donorbox.org/embed/okturtles-donation?language=en" name="donorbox" allowpaymentrequest="allowpaymentrequest" seamless="seamless" frameborder="0" scrolling="no" height="550px" width="100%" style="max-width: 500px; min-width: 250px; max-height:none!important" allow="payment"></iframe>
+          <div class="c-credit-block">
+            <h2 class="c-sub-title">Credit/Debit Card Donation</h2>
+            <a class="button is-success c-link-btn" href="https://donorbox.org/okturtles-donation" target="_blank">Donate with Donorbox</a>  
+          </div>
+          
+          <div class="c-credit-block">
+            <h2 class="c-sub-title">Others</h2>
+            <a class="button is-primary c-link-btn" href={resolvePath("/other-ways-to-support")}>Other ways to Donate</a>
+          </div>
         </div>
       </section>
-      <div class="c-container-center">
-        <a class="button is-primary" href={resolvePath("/other-ways-to-support")}>Other ways to Donate</a>
-      </div>
-      <section class="container c-grid">
+
+      <section class="container c-grid c-mb-6">
```

Comment:
Boy-scouting this mobile-bug:

<img src='https://github.com/user-attachments/assets/7cbd486f-9f9d-418e-af44-7677b637c14d' width='390'>